src/avs/AVSList.jsx

```diff
@@ -270,7 +267,7 @@ export default function AVSList() {
               state.avs &&
               [...Array(10 - state.avs.length)].map((_, i) => (
                 <TableRow key={i}>
-                  <TableCell className="h-[3.82rem] w-2/5"></TableCell>
+                  <TableCell className="w-2/5 p-5"></TableCell>
```

Comment:
when the list is not full, we should keep the empty rows same height so the table looks the same and no layout shift

<img width="548" alt="image" src="https://github.com/user-attachments/assets/64bac4f7-520b-4351-9bfa-3ca4a0439d7a">

Reply:
```suggestion
                  <TableCell className="w-2/5 py-8"></TableCell>
```
We can increase vertical padding to solve this.

Reply:
I'm not sure what are you trying to achieve with this. Currently, it looks like this. What issue do you see here?

![image](https://github.com/user-attachments/assets/36f4c70a-e5d8-45b6-b6bd-973b004ccd5d)